src/Symfony/Component/HttpKernel/Tests/Fixtures/BundleWithExplicitName/Bundle.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Symfony\Component\HttpKernel\Tests\Fixtures\BundleWithExplicitName;
+
+use Symfony\Component\HttpKernel\Bundle\Bundle as BaseBundle;
+
+class Bundle extends BaseBundle
```

Comment:
We could put this class and the `GuessedNameBundle` class into the `BundleTest.php` file.